sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -151,6 +153,10 @@ object GpuOrcScan extends Arm {
       .getOption("spark.sql.orc.mergeSchema").exists(_.toBoolean)) {
       meta.willNotWorkOnGpu("mergeSchema and schema evolution is not supported yet")
     }
+
+    // We need the value of 'isOrcFloatTypesToStringEnable' in 'canCast' method.
+    // So we get a reference of 'meta.conf'.
+    rapidsConf = meta.conf
```

Comment:
This is a WAR, not a good way.
Instead, pass the boolean value (isOrcFloatTypesToStringEnable, store as a local var) as a parameter throughout the call stack
```
  GpuOrcFileFilterHandler -> getOrcPartitionReaderContext -> checkSchemaCompatibility -> checkTypeCompatibility -> canCast
```